core/src/test/java/org/apache/struts2/views/jsp/ui/ComboBoxTest.java

```diff
@@ -73,6 +73,8 @@ public void testSimple() throws Exception {
         tag.setId("cb");
         tag.setList("collection");
 
+        stack.getActionContext().getSession().put("nonce", "r4nd0m");
```

Comment:
why is this needed?

Reply:
This is putting the  nonce value in the session, just like the `CspInterceptor` does. We use it to test if the script blocks are rendered with nonce attributes.